autogluon_zeroshot/contexts/context.py

```diff
@@ -0,0 +1,329 @@
+import time
+from typing import Dict, List
+from pathlib import Path
+
+import boto3
+from botocore.errorfactory import ClientError
+import pandas as pd
+
+from autogluon.common.loaders import load_pd
+from autogluon.common.utils.s3_utils import download_s3_files
+from autogluon.common.utils.s3_utils import is_s3_url, s3_path_to_bucket_prefix
+
+from .utils import load_zeroshot_input
+from ..loaders import load_configs, load_results, combine_results_with_score_val, Paths
+from ..simulation.simulation_context import ZeroshotSimulatorContext
+
+
+# TODO: Add logic to download files if they don't exist
+class BenchmarkPaths:
+    def __init__(self,
+                 result: str,
+                 results_by_dataset: str,
+                 raw: str,
+                 comparison: str,
+                 task_metadata: str = None,
+                 zs_pp: str = None,
+                 zs_gt: str = None,
+                 configs: List[str] = None,
+                 ):
+        self.result = result
+        self.results_by_dataset = results_by_dataset
+        self.raw = raw
+        self.comparison = comparison
+        self.task_metadata = task_metadata
+        self.zs_pp = zs_pp
+        self.zs_gt = zs_gt
+
+        if configs is None:
+            configs_prefix_1 = str(Paths.data_root / 'configs/configs_20221004')
```

Comment:
You dont need the string conversion at this place since you converting everything to string at l41. 
Also to make it portable we have to avoid manual '/'  e.g. 
```suggestion
            configs_prefix_1 = Paths.data_root / 'configs' / 'configs_20221004'
```

Reply:
Good catch! Updated.